### PR TITLE
Promethean Balancing, Lore-compliant Modification of Security Equipment

### PR DESCRIPTION
**Pull request by Mechoid**
Tweaks:
- Regeneration now takes approximately fifteen seconds, no longer full-heals internal organs. Functionally half-heals all internal organs instead.
- Species based slip resistance is based on foot coverage. If your feet are covered your natural traction and grip are replaced by that of the object covering them.
- Species with the regeneration power may have a hastened or slowed regeneration speed.
- Promethean Siemens raised from 0.3 to 0.4
- Promethean max age raised from 5 to 10, pending wikipage update.
- Xenotasers no longer ignore armor 100% on Prometheans. Agony scaled based on laser resistance of worn armor rather than siemens. 3 shots on an unarmored promethean to down them, same as a normal taser on a human.

Additions:
- Security biohazard lockers now contain a retrofitted xeno-taser due to the hostile takeover of the NRS Prometheus and Aetolus. Reference: https://puu.sh/zgXEU/8aebdd1cf2.png
- Security borgs now have the same xeno taser in addition to their normal taser, for the same reason.
- Security Xeno-tasers are in all ways inferior to the Research variants, to still have reason for Science to provide them in extreme situations. 10 shots from 20, 42 agony rather than 60, standard taser accuracy rather than the +2 used by Research.

### CODE REVIEW

**Comment by elgeonmb** [comment]
*cackles*

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -16,6 +16,8 @@
 	var/last_spit = 0 //Timestamp.
 
 	var/can_defib = 1	//Horrible damage (like beheadings) will prevent defibbing organics.
+	var/active_regen = 0 //Used for the regenerate proc in human_powers.dm
```
Comment (Anewbe) [inline code/modules/mob/living/carbon/human/human.dm]: If this is only one or zero, make it TRUE or FALSE?

**Comment by Mechoid** [comment]
Normal xenotasers down in two hits through any armor if they contact.
Realized it's probably smarter to put the age-bump here as well pending adjustment of lore dates, DNM and WIP regarding both.

File: code/game/objects/structures/crates_lockers/closets/l3closet.dm [hunk 2]
```diff
@@ -38,7 +38,7 @@
 	..()
 	new /obj/item/clothing/suit/bio_suit/security(src)
 	new /obj/item/clothing/head/bio_hood/security(src)
-
+	new /obj/item/weapon/gun/energy/taser/xeno/sec(src)
```
Comment (Anewbe) [inline code/game/objects/structures/crates_lockers/closets/l3closet.dm]: Weren't there two of these per locker?

File: code/modules/mob/living/carbon/human/species/station/prometheans.dm [hunk 3]
```diff
@@ -39,7 +39,7 @@ var/datum/species/shapeshifter/promethean/prometheans
 	virus_immune =	1
 	blood_volume =	560
 	min_age =		1
-	max_age =		5
+	max_age =		10
```
Comment (Anewbe) [inline code/modules/mob/living/carbon/human/species/station/prometheans.dm]: Has the wiki's timeline been adjusted for this?

Comment (Mechoid) [reply code/modules/mob/living/carbon/human/species/station/prometheans.dm]: This is the reason for DNM

File: code/game/objects/structures/crates_lockers/closets/l3closet.dm (hunk 2 above)
Comment (Mechoid) [reply code/game/objects/structures/crates_lockers/closets/l3closet.dm]: There were not, one per as one suit per.

File: code/modules/mob/living/carbon/human/species/station/prometheans.dm (hunk 3 above)
Comment (Mechoid) [reply code/modules/mob/living/carbon/human/species/station/prometheans.dm]: Timeline updated.

**Pull request merged**